cogs/commands/dalle.py

```diff
@@ -0,0 +1,66 @@
+import logging
+import discord
+import openai
+from discord.ext import commands
+from discord.ext.commands import Bot, Context, clean_content
+import aiohttp
+import asyncio
+from io import BytesIO
+
+from config import CONFIG
+
+LONG_HELP_TEXT = """
+Apollo is more creative than you think...
+
+Apollo can now generate imagefes using openAI's DALL-E model. 
+To use Apollo, simply type `!dalle <prompt>` . Apollo will then generate an image based on the prompt.
+"""
+
+SHORT_HELP_TEXT = "Apollo is more creative than you think..."
+
+chat_cmd = CONFIG.PREFIX + "dalle"
+
+def clean(msg, *prefixes):
+    for pre in prefixes:
+        msg = msg.strip().removeprefix(pre)
+    return msg.strip()
+
+
+class Dalle(commands.Cog):
+    def __init__(self, bot: Bot):
+        self.bot = bot
+        openai.api_key = CONFIG.OPENAI_API_KEY
+        self.cooldowns = {}
+
+    @commands.command()
+    async def dalle(self, ctx: Context, *, prompt: clean_content):
+        prompt = clean(prompt, chat_cmd, CONFIG.PREFIX)
```

Comment:
I dont think this call is needed, see [!say](https://github.com/UWCS/apollo/blob/master/cogs/commands/say.py) for how to get the rest of the message.